examples/annotation_import/text.ipynb

```diff
@@ -105,16 +105,9 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    print("   ")
</pre><p>Let's get rid of this print statement</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>